test/i18n.configureCookiename.js

```diff
@@ -2,7 +2,7 @@ var i18n = require('../i18n'),
   should = require("should"),
   path = require("path");
 
-describe('Locale switching should work queryParameter', function() {
+describe('Locale switching should work when set via cookie', function() {
```

Comment:
yap I've seen this... thanks again - I bet there are still some more typos in readme, comments & tests - 6 years of part time maintenance...